src/EventTimer.ts

```diff
@@ -66,8 +74,32 @@ export class EventTimer {
 		return this.init();
 	}
 
+	/**
+	 * Returns all commercial timers. CMP-related timers are not tracked
+	 * by EventTimer so they need to be concatenated to EventTimer's private events array.
+	 */
+	public get events(): Event[] {
+		return typeof window.performance !== 'undefined' &&
+			'mark' in window.performance
+			? [
+					...this._events,
+					...EventTimer._externallyDefinedEventNames
+						.map(
+							(eventName) =>
+								new Event(
+									eventName,
+									window.performance.getEntriesByName(
+										eventName,
+									)[0] ?? 0,
+								),
+						)
+						.filter((event) => event.ts > 0),
+			  ]
+			: this._events;
```

Comment:
Just wondering if this spreading logic is captured by any tests?

Reply:
Done now!